x-pack/packetbeat/Jenkinsfile.yml

```diff
@@ -13,6 +14,12 @@ when:
     tags: true                 ## for all the tags
 platform: "linux && ubuntu-18" ## default label for all the stages
 stages:
+# TODO confirm this works
```

Comment:
Does it work? :slightly_smiling_face:

Reply:
I have deleted the file is not used, @v1v has to confirm it

Reply:
It is not used indeed, but I left there in case it will be supported in the future, so we can delete it for simplicity

Reply:
I have to re-add the file because there is a magefile on the master branch, I've to check if we can backport this to all branches